### PR TITLE
Cherry pick docs

### PR DESCRIPTION
**Pull request by SvenDowideit**
documentation fixes we'll publish in in the next day or so.

### CODE REVIEW

**Comment by dongluochen** [comment]
#2205 is not pure doc update. It has code update depending on the change migrating from samalba/dockerclient to docker/engine-api.

**Comment by thaJeztah** [comment]
@dongluochen only the docs/markdown files are used from the docs branch, so you can ignore the code changes. But, can you verify if there's nothing in these cherry picks that's not yet in the current release? The idea is to do an update of docs.docker.com after this is merged, so it's important that there's nothing in here that doesn't work on the current release of Swarm

**Comment by dongluochen** [comment]
We release 1.2.2 this week with all the changes. I don't see any problem form the cherry picks. 

LGTM

**Comment by SvenDowideit** [comment]
excellent, thank you @dongluochen

**Comment by sfsmithcha** [comment]
docs updates LGTM

File: docs/secure-swarm-tls.md [hunk 1]
```diff
@@ -73,49 +73,49 @@ certificates to increase security.
 
 <!--[metadata]>Need to know about encryption too<![end-metadata]-->
 
-You can configure both the Docker Engine CLI and the Engine daemon to require
-TLS for authentication.  Configuring TLS means that all communications between
-the Engine CLI and the Engine daemon must be accompanied with, and signed by a
-trusted digital certificate. The Engine CLI must provide its digital certificate
-before the Engine daemon will accept incoming commands from it.
+You can configure both the Docker Egnine CLI and the Docker Engine daemon to require
```
Comment (allencloud) [inline docs/secure-swarm-tls.md]: Egnine -> Engine

**Pull request merged**